pkgs/development/python-modules/cryptography/default.nix

```diff
@@ -20,18 +20,17 @@
 }:
 
 buildPythonPackage rec {
-  # also bump cryptography_vectors
   pname = "cryptography";
-  version = "2.5";
+  version = "2.6.1"; # Also update the hash in vectors.nix
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "00c4d7gvsymlaw0r13zrm32dcnarmpayjyrh65yymlmr6mrbcij9";
+    sha256 = "19iwz5avym5zl6jrrrkym1rdaa9h61j20ph4cswsqgv8xg5j3j16";
   };
 
   outputs = [ "out" "dev" ];
 
-  buildInputs = [ openssl cryptography_vectors ]
+  buildInputs = [ openssl ]
```

Comment:
Should `openssl` be pointed towards `openssl_1_1`? At least for `sourcehut` to work support for `ed25519` is needed which isn't provided by the default `openssl`.

```
    File "/nix/store/jszvzjmq5nx90dlw8d80hcyfwzzqzj7p-python3.7-cryptography-2.6.1/lib/python3.7/site-packages/cryptography/hazmat/primitives/asymmetric/ed25519.py", line 63, in from_private_bytes
      _Reasons.UNSUPPORTED_PUBLIC_KEY_ALGORITHM
  cryptography.exceptions.UnsupportedAlgorithm: ed25519 is not supported by this version of OpenSSL.
```